Admin/AbstractAdmin.php

```diff
@@ -2816,6 +2659,163 @@ public function getDashboardActions()
     }
 
     /**
+     * {@inheritdoc}
+     */
+    protected function configure()
```

Comment:
This was on `AdminIntefarce`. I think it should be abstract to keep the same implementation logic.